rskj-core/src/test/java/co/rsk/peg/whitelist/constants/WhitelistConstantsTest.java

```diff
@@ -0,0 +1,64 @@
+package co.rsk.peg.whitelist.constants;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class WhitelistConstantsTest {
+
+    @ParameterizedTest
+    @MethodSource("getLockWhitelistChangeAuthorizerProvider")
+    void getLockWhitelistChangeAuthorizer(WhitelistConstants whitelistConstants, AddressBasedAuthorizer expectedLockWhitelistChangeAuthorizer) {
+        AddressBasedAuthorizer actualLockWhitelistChangeAuthorizer = whitelistConstants.getLockWhitelistChangeAuthorizer();
+
+        assertThat(actualLockWhitelistChangeAuthorizer, samePropertyValuesAs(expectedLockWhitelistChangeAuthorizer));
+    }
+
+    private static Stream<Arguments> getLockWhitelistChangeAuthorizerProvider() {
+        //MainNet
+        List<ECKey> mainNetLockWhitelistAuthorizedKeys = Arrays.stream(new String[]{
+            "041a2449e9d63409c5a8ea3a21c4109b1a6634ee88fd57176d45ea46a59713d5e0b688313cf252128a3e49a0b2effb4b413e5a2525a6fa5894d059f815c9d9efa6"
+        }).map(hex -> ECKey.fromPublicOnly(Hex.decode(hex))).collect(Collectors.toList());
```

Comment:
For these, it can be simplified as:

```java
List<ECKey> mainNetLockWhitelistAuthorizedKeys = Arrays.asList(       
    ECKey.fromPublicOnly(Hex.decode("041a...")));
```

Reply:
Done!